mmv1/products/workstations/WorkstationConfig.yaml

```diff
@@ -235,6 +236,13 @@ properties:
             name: 'disablePublicIpAddresses'
             description: |
               Whether instances have no public IP address.
+          - !ruby/object:Api::Type::Boolean
+            name: 'enableNestedVirtualization'
+            description: |
+              Whether to enable nested virtualization on the Compute Engine VMs backing the Workstations.
+
+              See https://cloud.google.com/workstations/docs/reference/rest/v1beta/projects.locations.workstationClusters.workstationConfigs#GceInstance.FIELDS.enable_nested_virtualization
+            default_from_api: true
```

Comment:
Based on the test logs, it looks like the API doesn't return a value for this field unless it's set, so I don't think we need `default_from_api`. Would it be possible to remove this?

```suggestion
```

Reply:
Sounds good - I've removed it. Thanks for the suggestion!